Applications/CLI/ogs.cpp

```diff
@@ -99,6 +100,14 @@ int main(int argc, char *argv[])
 		"output directory");
 	cmd.add(outdir_arg);
 
+	TCLAP::ValueArg<std::string> log_level_arg(
+		"l", "log-level",
```

Comment:
Do we need the shortcut `-l`? In general I'd not introduce them until OGS6 is kind of feature-complete.